locales/en-US/app.ftl

```diff
@@ -12,9 +12,16 @@ footerLinkPrivacy = Privacy
 footerLinkTerms = Terms
 footerLinkLegal = Legal
 footerLinkFeedback = Give Feedback
-footerLinkAbout = About Test Pilot
-
-## Items in the menu.
+footerLinkAboutHeader = About
+footerLinkAboutUs = About us
+footerLinkSupportHeader = Support
+footerLinkDownload = Download Firefox
+footerLinkMobile = Mobile
+footerLinkFeatures = Features
+footerLinkBeta = Beta, Nightly, Developer Edition
+footerLinkRetire = <h3>Uninstall Test Pilot</h3>
```

Comment:
Do we need to expose the HTML tags here? Can we just have the string?

Reply:
This is a typo, thanks for catching that